src/app/dashboard/learning-path/[id]/page.tsx

```diff
@@ -0,0 +1,206 @@
+        <div className="w-[2px] bg-line-secondary h-full absolute top-0 left-[18px]"></div>
+      )}
+
+      <div className="relative w-[40px] h-[46px] flex items-center justify-center text-white bg-background-secondary text-2xl font-semibold border border-secondary-border rounded-lg">
+        {index}
+      </div>
+
+      <div className="border border-secondary-border hover:border-primary-dark-border/30 transition-all rounded-lg flex max-h-[150px] w-full bg-background-secondary text-white group">
+        <img
+          src={`${process.env.STRAPI_BASE_URL}${data.attributes.cover_images.data.attributes.url}`}
+          className="w-[200px] object-cover rounded-tl-lg rounded-bl-lg flex-shrink-0"
+        />
+
+        <div className="p-4 flex flex-col justify-between">
+          <div>
+            <p className="font-semibold mb-2 group-hover:text-primary">{data.attributes.title}</p>
+            <p className="text-secondary-content-dark text-sm">
+              {data.attributes.short_descriptions}
+            </p>
+          </div>
+
+          <div className="flex items-center justify-between">
+            <div className="flex items-center gap-3">
+              <div className="flex items-center gap-2 text-secondary-content-dark">
+                <FaVideo className="w-4 h-4" />
+                <p className="text-xs">{data.attributes.total_lesson} Chapters</p>
+              </div>
+
+              <div className="flex items-center gap-2 text-secondary-content-dark">
+                <FaClock className="w-4 h-4" />
+                <p className="text-xs">{formatDuration(data.attributes.total_duration)}</p>
+              </div>
+            </div>
+
+            <DifficulityIcon type={data.attributes.difficulty} />
+          </div>
+        </div>
+      </div>
+    </Link>
+  )
+}
+
+const fetchCategoryFromStrapi = async (id: string) => {
+  const query = qs.stringify({
+    populate: "courses&select=name,icon,courses,courses.cover_images,mentors,mentors.avatar",
+    fields: "*",
+  })
+
+  const response = await fetch(`${process.env.STRAPI_BASE_URL}/api/categories/${id}?${query}`, {
+    cache: "no-store",
+  })
+  const strapiResponse: StrapiResponse<CategoryResponse> = await response.json()
+  return strapiResponse
+}
+
+const LearningPathDetail = async ({ params }: { params: { id: string } }) => {
+  const categoryResponse = await fetchCategoryFromStrapi(params.id)
+  const courseList = categoryResponse.data.attributes.courses.data
+  const mentorList = categoryResponse.data.attributes.mentors.data
+
+  return (
+    <div className="flex flex-col gap-12">
+      <div className="flex flex-col gap-6">
+        {/* Breadcrumb */}
+        <div className="flex items-center gap-2 text-xs">
+          <Link href="/dashboard/learning-path" className="text-secondary-content-dark">
+            Learning Path
+          </Link>
+          <p>/</p>
+          <p>{categoryResponse.data.attributes.title}</p>
+        </div>
```

Comment:
[not urgent] I've created breadcrumbs component in main. we might want to use that in the future.